docs/guide/algos.rst

```diff
@@ -19,7 +19,8 @@ TD3          ✔️          ❌            ❌                ❌
 
 
 .. note::
-    Non-array spaces such as ``Dict`` or ``Tuple`` are not currently supported by any algorithm.
+   ``Tuple`` spaces are not supported by any environment however ``Dict`` spaces of ``Box`` spaces are. 
+   ``Dict`` spaces of containing other kinds of spaces (e.g., ``Discrete``) have not yet been explored.
```

Comment:
why not?
we should have tests for that (using dummy dict env)

Reply:
I haven't had a moment to try other kinds of spaces yet but will try to this week using the dummy dict env. It will be a good test to see if frame stacking will work out of the box (pun intended).

Reply:
spoiler alert: it does not ^^"
but I did some bug fixes and now it should ;)